include/nbl/ext/DebugDraw/CDraw3DLine.h

```diff
@@ -120,7 +120,7 @@ class CDraw3DLine : public core::IReferenceCounted, public core::InterfaceUnmova
 		}
 		// If @fence is not nullptr, you'll get a new fence assigned to @fence that you can wait for,
 		// If @fence is nullptr, the function will automatically manage fence waiting
-        void updateVertexBuffer(video::IGPUQueue* queue, video::IGPUFence* fence = nullptr);
+        void updateVertexBuffer(video::IGPUQueue* queue, core::smart_refctd_ptr<video::IGPUFence>* fence = nullptr);
```

Comment:
well, kinda what I meant but you still don't write any fence to `fence` for user to be able to wait on it